src/suites/cts/validation/setBlendColor.spec.ts

```diff
@@ -0,0 +1,45 @@
+export const description = `
+setBlendColor validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+
+import { ValidationTest } from './validation_test.js';
+
+// TODO: Move beginRenderPass to a Fixture class.
```

Comment:
What do you mean by this? it's already in a fixture class - do you mean to move it up to a common file?

Reply:
Yup. Fixed.